src/cartesi-machine.lua

```diff
@@ -402,7 +402,7 @@ local memory_range_replace = {}
 local ram_image_filename = images_path .. "linux.bin"
 local ram_length = 64 << 20
 local rom_image_filename = images_path .. "rom.bin"
-local rom_bootargs = "console=hvc0 rootfstype=ext2 root=/dev/mtdblock0 rw quiet swiotlb=noforce"
+local rom_bootargs = "console=hvc0 rootfstype=ext2 root=/dev/mtdblock0 rw quiet swiotlb=noforce init=/opt/cartesi/bin/init"
```

Comment:
There is a test in `machine-test.lua` that will also need this change.

Reply:
Nice catch, adding it to: `src/tests/machine-test.lua:512`

Reply:
I'll need an updated RootFS for this to pass CI, it is in the works.